core/src/main/scala/org/bitcoins/core/script/interpreter/ScriptInterpreter.scala

```diff
@@ -436,10 +436,46 @@ sealed abstract class ScriptInterpreter extends BitcoinSLogger {
       else evaluated
     }
 
+    def rebuildV0(witness: ScriptWitness, program: Seq[ScriptToken]): Either[
+      (Seq[ScriptToken], ScriptPubKey),
```

Comment:
Ok, so since we are touching this, I think we should also do `Either[ScriptError,(Seq[ScriptToken],ScriptPubKey)]` (right-bias) return type as in Scala 2.12 they made `Either` right-biased